docs/contributing/coding-guidelines/ros-nodes/parameters.md

```diff
@@ -1,5 +1,144 @@
 # Parameters
 
-!!! warning
+## Overview
 
-    Under Construction
+Parameters in ROS are associated with individual nodes. Parameters are used to configure nodes at startup (and during runtime), without changing the code. The lifetime of a parameter is tied to the lifetime of the node (though the node could implement some sort of persistence to reload values after restart).<br>
+Parameters are addressed by node name, node namespace, parameter name, and parameter namespace. Providing a parameter namespace is optional.<br>
+Each parameter consists of a key, a value, and a descriptor. The key is a string and the value is one of the following types: bool, int64, float64, string, byte[], bool[], int64[], float64[] or string[]. By default all descriptors are empty, but can contain parameter descriptions, value ranges, type information, and additional constraints.<br>
+For an hands-on tutorial with ROS parameters [see here](https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Understanding-ROS2-Parameters/Understanding-ROS2-Parameters.html)
+
+## Parameter commands
+
+We can use 'ros2 param -h' to show all the commands about parameter.
+
+- Commands:
+  - delete &emsp;&ensp;&thinsp; Delete parameter
+  - describe &ensp;&thinsp; Show descriptive information about declared parameters
+  - dump &emsp;&emsp; Dump the parameters of a node to a yaml file
+  - get &emsp;&emsp;&emsp; Get parameter
+  - list &emsp;&emsp;&emsp;&thinsp; Output a list of available parameters
+  - load &emsp;&emsp;&thinsp;&thinsp;&thinsp; Load parameter file for a node
+  - set &emsp;&emsp;&emsp;&thinsp; Set parameter
+
+## Declaring parameters
+
+By default, a node needs to declare all of the parameters that it will accept during its lifetime. This is so that the type and the name of the parameter are well-defined at node startup time, which reduces the chances of misconfiguration later on.<br>
+For example,we can use the 'declare_parameter' to declare and set initial parameter values in the code.
+
+```c
+class MinimalParam : public rclcpp::Node
+{
+public:
+    MinimalParam(): Node("minimal_param_node")
+    {
+        this->declare_parameter("my_parameter", "world");
+        timer_ = this->create_wall_timer(
+            1000ms, std::bind(&MinimalParam::timer_callback, this));
+    }
+}
+```
```

Comment:
Same here.
https://docs.ros.org/en/humble/Tutorials/Beginner-Client-Libraries/Using-Parameters-In-A-Class-CPP.html